src/style.css

```diff
@@ -1,3 +1,80 @@
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+}
+
+/* CONTAINER */
+
+.container {
+  display: flex;
+  justify-content: space-between;
+  padding: 0 50px;
+  align-items: center;
+}
+
+/* LOGO */
+
+.home__link {
+  display: flex;
+  align-items: center;
+}
+
+.image {
+  width: 40px;
+  height: 40px;
+}
+
+/* LINKS */
+
+.links {
+  display: flex;
+  list-style: none;
+  padding: 0;
+  margin: 0;
+}
+
+.nav__link {
+  margin-left: 20px;
```

Comment:
Be consistent with your vertical margins. Here you are adding left margin, which is horizontal. If you need to add vertical space, consider using padding or adjusting the line-height.